src/Raven.Client/ServerWide/Operations/Logs/GetLogsConfigurationOperation.cs

```diff
@@ -59,8 +60,18 @@ public class GetLogsConfigurationResult
         public bool UseUtcTime { get; set; }
 
         /// <summary>
-        /// Logs retention time
+        /// Logs retention time (null if RetentionTime is TimeSpan.MaxValue)
```

Comment:
TimeSpan is struct so it can't be null, so I'm affraid comment here is misleading.

Reply:
done